ddtrace/internal/_encoding.pyx

```diff
@@ -9,15 +9,14 @@ DEF MSGPACK_ARRAY_LENGTH_PREFIX_SIZE = 5
 
 
 cdef extern from "Python.h":
-    char* PyUnicode_AsUTF8AndSize(object obj, Py_ssize_t *l) except NULL
+    const char* PyUnicode_AsUTF8(object o)
```

Comment:
is this already available with `from cpython cimport *` ?

Reply:
Nope unfortunately I ran into import errors before manually importing this, and I read in documentation somewhere that:

> Cython doesn't wrap the whole CPython's C-API in cpython.*-headers, only the most important part. If a function is not wrapped, it can be always done on the fly via ...